packages/ui-library/src/icons/lang.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { CustomSvgIcon, SvgIconProps } from '../components/svgicon';
+
+interface IconProps extends SvgIconProps {
+  size?: string;
+}
+
+export default function LangIcon({ size, ...props }: IconProps) {
+  return (
+    <CustomSvgIcon viewBox="0 0 25 25" style={size ? { fontSize: size } : {}} {...props}>
```

Comment:
You've checked that the viewbox matches the one from your initially exported svg right?

Reply:
out of sheer luck, it did 🥶